engine/platform/psvita/sys_psvita.c

```diff
@@ -75,32 +76,76 @@ const size_t __vrtld_num_exports = sizeof( aux_exports ) / sizeof( *aux_exports
 
 /* end of export crap */
 
+static const char *PSVita_GetLaunchParameter( char *outbuf )
+{
+	SceAppUtilAppEventParam param;
+	memset( &param, 0, sizeof( param ) );
+	sceAppUtilReceiveAppEvent( &param );
+	if( param.type == 0x05 )
+	{
+		sceAppUtilAppEventParseLiveArea( &param, outbuf );
+		return outbuf;
+	}
+	return NULL;
+}
+
 void Platform_ShellExecute( const char *path, const char *parms )
 {
 	Con_Reportf( S_WARN "Tried to shell execute ;%s; -- not supported\n", path );
 }
 
+/*
+===========
+PSVita_GetArgv
+
+On the PS Vita under normal circumstances argv is empty, so we'll construct our own
+based on which button the user pressed in the LiveArea launcher.
+===========
+*/
+int PSVita_GetArgv( char ***out_argv )
+{
+	static const char *fake_argv[MAX_ARGV] = { "", NULL };
```

Comment:
argv[0] shouldn't be an empty string. You can write anything you want there, just as a fake executable name.

Reply:
It is an empty string in the actual `argv` when the game is started normally, so I wanted it to act the same in both cases.